src/functionalTest/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefFunctionalTest.java

```diff
@@ -322,11 +338,11 @@ public void shouldUploadXlsFileSuccessfully() throws IOException {
     @ToggleEnable(mapKey = CASEWORKER_FILE_UPLOAD, withFeature = true)
     @ExtendWith(FeatureToggleConditionExtension.class)
     @Order(1)
-    public void shouldUploadServiceRoleMappingXlsxFileSuccessfully() throws IOException {
+    void shouldUploadServiceRoleMappingXlsxFileSuccessfully() throws IOException {
         ExtractableResponse<Response> uploadCaseWorkerFileResponse =
                 uploadCaseWorkerFile("src/functionalTest/resources/ServiceRoleMapping_BBA9.xlsx",
                         200, IDAM_ROLE_MAPPINGS_SUCCESS, TYPE_XLS,
-                        ROLE_CWD_ADMIN);
+                        ROLE_CWD_ADMIN,Boolean.TRUE);
```

Comment:
you can set this to false as for service to idam mapping file there is no need to update version